projects/core/src/lib/components/home-page/home-page.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="home">
+	<section class="about">
+		<h2 class="home__title">What is JS Middle Developer Growth ?</h2>
+		<p class="about__content">
+			It's educational initiative which hepls candidate to become D2 quickly and with better
+			quality
+		</p>
+	</section>
+	<hr class="home__hr" />
```

Comment:
it's not a good idea to name an element by its tag name. please rename according to context of using

Reply:
done